bin/src/command/orders.rs

```diff
@@ -658,15 +677,24 @@ impl CommandServer {
       })
     );
 
+    // Remove the backend or frotend from the master configuration
     match order {
-      ProxyRequestData::AddBackend(_) => self.backends_count += 1,
-      ProxyRequestData::RemoveBackend(_) => self.backends_count -= 1,
-      ProxyRequestData::AddHttpFront(_) | ProxyRequestData::AddHttpsFront(_) | ProxyRequestData::AddTcpFront(_) => {
-        self.frontends_count += 1;
-      },
-      ProxyRequestData::RemoveHttpFront(_) | ProxyRequestData::RemoveHttpsFront(_) | ProxyRequestData::RemoveTcpFront(_) => {
-        self.frontends_count -= 1;
-      },
+      ProxyRequestData::RemoveBackend(ref mut backend) => self.config.applications.get_mut(&backend.app_id).map(|app| app.remove_backend(&backend.address)),
```

Comment:
why modify `self.config`? This is an object created from the configuration file, the real configuration state is in `self.state`